KarhooUISDKTests/TestCases/Screens/CheckoutScreen/KarhooCheckoutPresenterSpec.swift

```diff
@@ -94,6 +87,7 @@ class KarhooCheckoutPresenterSpec: XCTestCase {
      * And: Analytics event should fire
      */
     func testAdyenRequestCarAuthenticated() {
```

Comment:
The description for this test says _no_ metadata injected, yet it tests for XCTAssert_NotNil_(mockTripService.tripBookingSet?.meta)

Reply:
I believe it test if metadata are created at all. If you think business check should be updated - I encourage you to add proper changes